tests/yamls/resources.yaml

```diff
@@ -0,0 +1,120 @@
+apiVersion: apps/v1
```

Comment:
Can you a test file with multiple deployments/service/other resources?

Reply:
updated